### DISCUSSION TITLE
[11 gennaio 2022 - B] Esercizio 5

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#203] [Esami]
**Esercizio 5 (4 punti)**

Analizzare la macchina a stati mostrata in figura. Scrivere le tabelle degli stati futuri e di uscita e
disegnare l’automa (il diagramma di transizione degli stati).

![image](https://user-images.githubusercontent.com/69389461/211830518-22f80086-e179-46b3-b12f-5a0af30a4769.png)

**Comment by Elia-Belli** [comment]
![DS](https://user-images.githubusercontent.com/107944476/212292458-19eb3fad-4f47-4bff-b9b9-039d88aba89c.jpg)

**Comment by fraxb04** [comment]
@Elia-Belli 
corretto
